src/main/scala/dsptools/DspTester.scala

```diff
@@ -95,25 +95,29 @@ class DspTester[T <: Module](c: T,
 
   // [stevo]: poke a value in type typ to a UInt input
   //scalastyle:off cyclomatic.complexity
-  def dspPokeAs[T<:Data](bundle: Data, value: Double, typ: T): Unit = {
+  def dspPokeAs[TT<:Data](bundle: Data, value: Double, typ: TT): Unit = {
```

Comment:
Maybe instead of `TT` (which to me implies some sort of connection with `T` we should go with `U` or `A` or something